apps/design-system-docs/src/components/link-card/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { Card, CardBody, CardMedia } from '@wordpress/components';
+import { useId } from 'react';
+import defaultImage from './default.png';
+import styles from './style.module.scss';
+
+/**
+ * A card that links to a page.
+ */
+export const LinkCard = ( {
+	className,
+	href,
+	label,
+	description,
+	image = defaultImage,
+}: {
+	className?: string;
+	href: string;
+	label: string;
+	description?: string;
+	/**
+	 * Optional image to display in the card.
+	 * Should only be decorative, as it will be hidden from screen readers.
+	 */
+	image?: string;
```

Comment:
Will these replaced with previews of the component? Could we do live previews instead of static images?

Reply:
No, these will be "abstracted" thumbnails (DS-155), since these index items are general web UI concepts and not referring to any specific component (see comments on DS-157 for more context).